src/test/spec/auth.rs

```diff
@@ -84,7 +78,11 @@ async fn run_auth_test(test_file: TestFile) {
                     None => assert!(options.credential.is_none(), "{}", test_case.description),
                 }
             }
-            Err(_) => assert!(!test_case.valid, "{}", test_case.description),
+            Err(e) => assert!(
```

Comment:
While debugging, I found that it helped to see the actual error that occurred when we didn't expect one.